tests/nightly/dist_sync_kvstore.py

```diff
@@ -398,18 +432,25 @@ def check_trainer_kv_type(stype, grad_stype, update_on_kv):
     parser.add_argument('--no-gpu', dest='gpu', action='store_false')
     parser.add_argument('--no-multiprecision', dest='multiprecision', action='store_false')
     opt = parser.parse_args()
-    if opt.type == 'gluon':
-        test_gluon_trainer()
-    if opt.type == 'invalid':
+    if opt.type == 'gluon_type_cpu':
+        test_gluon_trainer_type()
+        exit()
+    if opt.type == 'gluon_step_cpu':
+        test_gluon_trainer_step()
+        exit()
+    if opt.type == 'gluon_sparse_step_cpu':
+        test_gluon_trainer_sparse_step()
+        exit()
+    if opt.type == 'invalid_cpu':
         test_invalid_operations()
+        exit()
     if opt.type == 'all' or opt.type == 'init':
         test_sync_init(opt.gpu)
+    kv = init_kv()
     if opt.type == 'all' or opt.type == 'default':
-        kv = init_kv()
         kv = set_optimizer(use_multiprecision=opt.multiprecision)
         test_sync_push_pull(opt.nrepeat)
-    # dont run non compressed tests after this as kvstore compression will be set here
+    # don't run non compressed tests after this as kvstore compression will be set here
     if opt.type == 'all' or opt.type == 'compressed':
```

Comment:
Could you remove this all parameter now?

Reply:
why remove it?

Reply:
Because in the current script, it no longer represents all tests.